modules/dream2nix/WIP-python-pdm/default.nix

```diff
@@ -9,14 +9,20 @@
   };
 
   libpyproject = import (dream2nix.inputs.pyproject-nix + "/lib") {inherit lib;};
-
-  selectWheel = import ../../flake-parts/lib/internal/python/selectWheel.nix {
+  libpyproject-fetchers = import (dream2nix.inputs.pyproject-nix + "/fetchers") {
     inherit lib;
-    pep599 = libpyproject.pep599;
-    python = config.deps.python3;
-    stdenv = config.deps.stdenv;
+    pkgs = config.deps;
```

Comment:
I think if we pass the whoel config.deps here, this will make the error in case of missing attributes more obscure. I created a patch for pyproject.nix here: https://github.com/nix-community/pyproject.nix/pull/29/files